### PR TITLE
Allow to set user's note

### PR DESCRIPTION
**Pull request by ilpianista**
See [Users API](https://docs.gitlab.com/ce/api/users.html#user-creation)

### CODE REVIEW

File: users.go [hunk 1]
```diff
@@ -188,6 +188,7 @@ type CreateUserOptions struct {
 	SkipConfirmation    *bool   `url:"skip_confirmation,omitempty" json:"skip_confirmation,omitempty"`
 	External            *bool   `url:"external,omitempty" json:"external,omitempty"`
 	PrivateProfile      *bool   `url:"private_profile,omitempty" json:"private_profile,omitempty"`
+	Note                *string `url:"note" json:"note,omitempty"`
```
Comment (svanharmelen) [inline users.go:191]:
```suggestion
	Note                *string `url:"note,omitempty" json:"note,omitempty"`
```

File: users.go [hunk 2]
```diff
@@ -231,6 +232,7 @@ type ModifyUserOptions struct {
 	SkipReconfirmation *bool   `url:"skip_reconfirmation,omitempty" json:"skip_reconfirmation,omitempty"`
 	External           *bool   `url:"external,omitempty" json:"external,omitempty"`
 	PrivateProfile     *bool   `url:"private_profile,omitempty" json:"private_profile,omitempty"`
+	Note               *string `url:"note" json:"note,omitempty"`
```
Comment (svanharmelen) [inline users.go:235]:
```suggestion
	Note               *string `url:"note,omitempty" json:"note,omitempty"`
```

**Review by svanharmelen — CHANGES_REQUESTED** [review]
Thanks @ilpianista... But seems you missed the `omitempty` option for the `url` tags.

**Comment by ilpianista** [comment]
Ops, thank you @svanharmelen

**Review by svanharmelen — APPROVED** [review]
Thanks @ilpianista! LGTM!

**Pull request merged**